eio/tls_eio.ml

```diff
@@ -29,7 +29,7 @@ module Raw = struct
       raise exn
 
   let write_t t cs =
-    try Flow.copy (Flow.cstruct_source [cs]) t.flow
+    try Flow.write t.flow [cs]
```

Comment:
is this really part of `improve fuzz test`? or a performance improvement? or a semantic change?

Reply:
It at least deserves a separate commit with a slightly more descriptive commit message.

Reply:
It's mostly just a simplification. It could be an optimisation if the underlying flow implements `write` directly. Otherwise, `write` just falls back to doing a `copy`, as in the old code. I can remove it if you prefer.

Reply:
As said, a separate commit with a commit message that includes your above description would be sufficient for me. I've disliked myself in the past to sneak any library changes in "update tests" commits, not being able to properly track years later why something changed in one way or another.